src/ccd2rgb.jl

```diff
@@ -0,0 +1,40 @@
+"""
+    ccd2rgb(red::ImageHDU, green::ImageHDU, blue::ImageHDU; stretch::String="linear", shape_out = size(red))
+
+Converts 3 grayscale ImageHDU into RGB by reprojecting them.
+
+# Arguments
+- `red`: Red channel data.
+- `green`: Green channel data.
+- `blue`: Blue channel data.
+- `stretch`: Stretch function applied. Can be linear, log, sqrt or ashinh.
+- `shape_out`: Shape of output RGB image.
+"""
+function ccd2rgb(red::ImageHDU, green::ImageHDU, blue::ImageHDU; stretch::String="linear", shape_out = size(red))
```

Comment:
Don't use a string for the `stretch` argument, use a function itself.  The operation applied is always `f(x)/x`?  Then let `f` be a function, default to `identity` and in the body do
```julia
I = map(x -> f(x)/x, I)
```
without even having an `if`.  The user will be even able to pass any function.  Also, I think that broadcast would work the same, right?  Then you can do
```julia
I .= (x -> f(x)/x).(I)
```
which will be an in-place operation